tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# noqa: D104
```

Comment:
What is the reason for needing this? I'm guessing it is the docstring for the module - usually I just put `"tests for the pollen operator"` or something like that

Reply:
Good point, have updated.